PickBus/Views/RouteListView/RouteListViewController.swift

```diff
@@ -170,21 +170,14 @@ final class RouteListViewController: UIViewController {
             // 수정할 부분 값이 nil인 경우 처리
             guard let nodeIndex = nodeIdDic[response[0].nodeid] else { fatalError() }
             for busIndex in busArray[nodeIndex].indices {
-                guard let myRouteNo = busArray[nodeIndex][busIndex].routeNo else {
-                    // 정보 없음 시, 뒤돌아가면 에러 발생해서 주석 처리.
-//                    fatalError()
-                    return
-                }
-                if let fetchBusInfo = response.filter({
-                    $0.routeno.stringValue == String(myRouteNo)
-                }).first {
-                    // 버스 정보가 있음
+                guard let myRoteId = busArray[nodeIndex][busIndex].routeId else { fatalError() }
```

Comment:
```suggestion
                guard let myRoteId = busArray[nodeIndex][busIndex].routeId else { return }
```
해당 부분은 fatalError로 할 시, 리스트 들어가자 마자 정보 받아오는 중에 뒤로가기 누르면 앱이 꺼지는 현상이 발생합니다. return으로 해두면 앱이 꺼지지 않습니다.

Reply:
그렇군요! 해당 부분 확인해보고 수정하겠습니다

Reply:
수정했습니다!